libvast_test/vast/test/fixtures/events.hpp

```diff
@@ -63,7 +65,13 @@ struct events {
   }
 
   std::vector<table_slice_ptr> copy(std::vector<table_slice_ptr> xs);
+
+  auto take_n(const std::vector<table_slice_ptr>& xs, size_t n) {
```

Comment:
How about calling that function simply `take`? The `n` parameter seems to be sufficient to convey that it's about multiple slices.